src/app/views/NoteCreate.svelte

```diff
@@ -169,6 +241,10 @@
   })
 
   $: loading = editor?.storage.fileUpload.loading
+
+  $: dvmPubkey = $handlers[0].pubkey
```

Comment:
This is crashing if handlers isn't defined. I mentioned this above, but let's just hardcode the dvm pubkey.

Reply:
I suggest to define the DVM pubkey in .env, some people seems to be mad at Shipyard

Hardcoding for now